client/src/handlers/handlers.ts

```diff
@@ -258,6 +258,27 @@ async function getJVMCmd(
   return cmd
 }
 
+/**
+ * Handle the user changing the active editor, to view a different file.
+ *
+ * If the new file is not part of the project, it shows a message to the user.
+ */
+export function handleChangeEditor(editor: vscode.TextEditor | undefined) {
+  if (editor === undefined) {
+    return
+  }
+
+  const isFlixFile = editor.document.uri.path.endsWith('.flix')
```

Comment:
Do we now for sure that document.uri is never undefined? Ditto for uri.path?

I am thinking if you open a new file/tab or some downloaded file etc.

Reply:
@sockmaster27 Ping :)

Reply:
New tab works, the `uri.path` is just something like "Untitled-1".
What do you mean by a downloaded file? Flix' release notes appear to work fine and aren't registered as a change-of-editor event.